### PR TITLE
Bug 1739407: Enhance host status to beter derive errors from operational status

### PR DESCRIPTION
**Pull request by jtomasek**
https://bugzilla.redhat.com/show_bug.cgi?id=1739407

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/constants.ts [hunk 1]
```diff
@@ -34,8 +35,8 @@ export const HOST_STATUS_TITLES = {
   [HOST_STATUS_DEPROVISIONING]: 'Deprovisioning',
   [HOST_STATUS_MAKING_HOST_AVAILABLE]: 'Making host available',
   [HOST_STATUS_VALIDATION_ERROR]: 'Validation Error(s)',
-  [HOST_STATUS_REGISTRATION_ERROR]: 'Registration Error',
-  [HOST_STATUS_PROVISIONING_ERROR]: 'Provisioning Error',
+  [HOST_STATUS_REGISTRATION_ERROR]: 'Registration error',
+  [HOST_STATUS_PROVISIONING_ERROR]: 'Provisioning error',
```
Comment (knowncitizen) [inline frontend/packages/metal3-plugin/src/constants.ts]: Why lower case?

Comment (jtomasek) [reply frontend/packages/metal3-plugin/src/constants.ts]: It resembles the wireframes and the convention used across the states

Comment (knowncitizen) [reply frontend/packages/metal3-plugin/src/constants.ts]: OK :)

**Pull request merged**